### PR TITLE
Update to Electron 1.3.2

### PR DESCRIPTION
**Pull request by miniak**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

https://github.com/electron/electron/releases/tag/v1.3.2

### CODE REVIEW

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -3763,6 +3835,29 @@ declare namespace Electron {
...
+		 */
+		setZoomLevel(level: number): void;
+		/**
+		 * Sends a request to get current zoom level.
+		 */
+		getZoomLevel(callback: (zoomLevel: number) => void): void;
+		/**
+		 * Sets the maximum and minimum zoom level.
+		 */
+		setZoomLevelLimits(minimumLevel: number, maximumLevel: number): void;
```
Comment (rhysd) [inline github-electron/github-electron.d.ts]: Just confirm: These zoom related APIs looks not included in v1.3.2 release.  They are missing in previous update, right?

**Comment by rhysd** [comment]
Nice work, LGTM :+1:

File: github-electron/github-electron.d.ts (hunk 1 above)
Comment (miniak) [inline github-electron/github-electron.d.ts]: They might be missing in the release notes. I did `git diff v1.3.1..v1.3.2 docs/api`. I will check and let them now to update the notes.

Comment (rhysd) [inline github-electron/github-electron.d.ts]: I got it.  I think documentation is more accurate than release note.

Comment (miniak) [inline github-electron/github-electron.d.ts]: Reported here: https://github.com/electron/electron/issues/6763.

**Pull request merged**